test/framework/openshiftrequestsutils.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
```

Comment:
Can these functions be moved to openshift.go to be with other functions dealing with OpenShift resources?

Reply:
I wanted to avoid that the other OpenShift resources file was too big... And this set of functions are related to only resources requests and limits. 
What do you think about this?

Reply:
ok, the openshift.go is big enough already.
Just please rename this file to reflect its content, for example to something like `openshift_resources.go`

Reply:
Renamed.